### ISSUE TITLE
Primereact Dropdown accessibility.  Value isn't being read by VoiceOver screen reader.

### ISSUE THREAD

**Issue by sparky742** [primefaces/primereact#5836]
### Describe the bug

On the Primereact Dropdown component, the value is not read by a screenreader.  However, this is working correctly in PrimeNG Dropdown.  This can be seen with Primereact 10.3.3 Basic Dropdown on https://primereact.org/dropdown/ 

macOS 14.3/Safari or Chrome

### Reproducer

https://primereact.org/dropdown/

### PrimeReact version

10.3.3

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. Navigate to https://primereact.org/dropdown/
2. Turn on voiceover, tab to Dropdown basic and change the value with the arrow keys and select
3. Value is not read
4. Tab out and back. Value is not read.

### Expected behavior

Value should be read when Dropdown is activated and voiceover voice reader is on.

**Comment by melloware** [comment]
@sparky742 can you check PrimeVue's does it read OK?  https://primevue.org/dropdown/

Let me know.

**Comment by sparky742** [comment]
> @sparky742 can you check PrimeVue's does it read OK? https://primevue.org/dropdown/
> 
> Let me know.

I just checked the 'Basic' example and that reads a selection perfectly.

**Comment by melloware** [comment]
OK fixing it for 10.5.2 but it really needs an overhaul to be like PrimeVue's dropdown component.
